pages/thankyou.js

```diff
@@ -26,13 +26,24 @@ export default function Confirmation(props) {
 
   // Send email to /thank-you api
   const handleResend = async () => {
-    const response = await fetch("/api/thank-you", {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify({ email: email }),
-    });
+    let response;
+    if (referrer === "unsubscribe") {
+      response = await fetch("/api/unsubscribe", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ email: email }),
+      });
+    } else {
+      response = await fetch("/api/thank-you", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ email: email, ref: referrer }),
```

Comment:
You can remove the ref parameter in the body, since the api does nothing with it.

Reply:
> I kind of thought since you created the /thank-you api route, the logic was going to be handled there, but it works here as well. Just a small thing to clean up and should be good to merge.

Nice catch, I was going to handle the logic in the /thank-you api, then I notice /unsubscribe api already handles it, so I decided to just use that.